app/views/foreman/unattended/snippets/_chef_client_omnibus_bootstrap.erb

```diff
@@ -0,0 +1,10 @@
+# install.sh will install latest chef client version
+curl -L https://www.chef.io/chef/install.sh | bash
```

Comment:
Quick install instructions from https://www.chef.io/download-chef-client/ :-) But I'll add more ways to bootstrap, either using packages from https://packagecloud.io/chef/stable or from distros where available. This will need more testing though and out of the scope of this PR.